cryptus-app/components/wallet_manager/wallet_manager.module.scss

```diff
@@ -0,0 +1,63 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+
+  display: flex;
+  width: min(100vw, v.$mobile);
+  margin: auto;
+  flex-direction: column;
+
+  background-color: v.$bg-color;
+  color: v.$text-color;
+}
+
+.header {
+  flex: 1 1 auto;
+
+  display: flex;
+  position: fixed;
+  top: 0;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+
+  width: min(100vw, v.$mobile);
+  height: v.$header-h;
+}
+
+.settingsTitle {
+  flex: 1 1 auto;
+
+  display: flex;
+  font-size: v.$font-size-ml;
+  justify-content: center;
+}
+
+.blankButton {
+  width: 24px;
+  background-color: black;
+  padding-right: 12px;
+}
+
+.line {
+  width: 75%;
```

Comment:
why %

Reply:
Because the line should take 75% of screen.